apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -42,7 +43,9 @@ const StopPageRedesign = ({
       <StopPageHeaderRedesign stop={stop} routes={routes} />
       <div className="container">
         <Alerts alerts={alerts} />
-        <div className="stop-routes-and-map">
+        {/* this is the mobile version */}
+        <StopPageOverlay routes={routes} stop={stop} schedules={schedules} />
+        <div className="stop-routes-and-map xs-hide">
```

Comment:
question (non-blocking): Why hide this on xs?

Reply:
Remnant of a different approach, will remove